src/CMakeLists.txt

```diff
@@ -4,9 +4,12 @@ project(chia-log-analysis VERSION 0.5)
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
+set(CMAKE_CXX_FLAGS "-lstdc++fs")  
```

Comment:
This flag should only be set when the compiler is GCC and the GCC version is 8.x, otherwise it is passed to other compilers that report warnings.